frontend_tests/node_tests/poll_widget.js

```diff
@@ -8,6 +8,8 @@ const {run_test} = require("../zjsunit/test");
 const blueslip = require("../zjsunit/zblueslip");
 const $ = require("../zjsunit/zjquery");
 
+const PollData = zrequire("../../static/shared/js/poll_data").default;
```

Comment:
Our convention is to omit `.default`, and I believe this is a relic of trying to get this to work?

Reply:
Switched from `export default class` to `export class` as discussed, done.